src/go/k8s/kind.yaml

```diff
@@ -11,3 +11,9 @@ nodes:
   image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
 - role: worker
   image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
+- role: worker
+  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
+- role: worker
+  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
+- role: worker
+  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
```

Comment:
I see one of the e2e test deletes one kind worker node and it creates 5 node cluster. I feel like it will destabilize our CI execution.

Reply:
Most tests rely on 3 worker nodes, this test just removes 2 of them so i needed to add 2, this could be an issue later if we test more nodes, but in that case I would suggest then that we move the test to a level up, something like e2e-v2-decommission